homeassistant/components/omnilogic/light.py

```diff
@@ -0,0 +1,225 @@
+"""Platform for light integration."""
+import logging
+import time
+
+from omnilogic import LightEffect, OmniLogicException
+import voluptuous as vol
+
+from homeassistant.components.light import ATTR_EFFECT, SUPPORT_EFFECT, LightEntity
+from homeassistant.helpers import entity_platform
+
+from .common import OmniLogicEntity, OmniLogicUpdateCoordinator
+from .const import COORDINATOR, DOMAIN
+
+SERVICE_SET_V2EFFECT = "set_v2_lights"
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
```

Comment:
Not really required, but it would be nice if we could type this method.

```suggestion
async def async_setup_entry(
    hass: HomeAssistant,
    entry: ConfigEntry,
    async_add_entities: AddEntitiesCallback,
) -> None:
```

Reply:
While we're in the patient... no problem.